keyboards/handwired/alcor_dactyl/keymaps/default/keymap.c

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 Ethan (@rocketstrong)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+/*
+          ┌──┐┌──┐┌──┐┌──┐┌──┐
+┌────┐┌──┐│  ││  ││  ││  ││  │
+│    ││  │└──┘└──┘└──┘└──┘└──┘
+└────┘└──┘┌──┐┌──┐┌──┐┌──┐┌──┐
+┌────┐┌──┐│  ││  ││  ││  ││  │
+│    ││  │└──┘└──┘└──┘└──┘└──┘
+└────┘└──┘┌──┐┌──┐┌──┐┌──┐┌──┐
+┌────┐┌──┐│  ││  ││  ││  ││  │
+│    ││  │└──┘└──┘└──┘└──┘└──┘
+└────┘└──┘┌──┐┌──┐┌──┐┌──┐
+┌────┐┌──┐│  ││  ││  ││  │
+│    ││  │└──┘└──┘└──┘└──┘
+└────┘└──┘┌──┐┌──┐
+  ┌──┐┌──┐│  ││  │  ┌──┐ ┌──┐
+  │  ││  │└──┘└──┘  │  │ │  │
+  └──┘└──┘          │  │ │  │
+                    │  │ └──┘
+                    └──┘ ┌──┐ ┌──┐
+                         │  │ │  │
+                         └──┘ └──┘
+                         ┌──┐┌──┐
+                         │  ││  │
+                         └──┘└──┘
+
+        ┌──┐┌──┐┌──┐┌──┐┌──┐
+        │  ││  ││  ││  ││  │┌──┐┌────┐
+        └──┘└──┘└──┘└──┘└──┘│  ││    │
+        ┌──┐┌──┐┌──┐┌──┐┌──┐└──┘└────┘
+        │  ││  ││  ││  ││  │┌──┐┌────┐
+        └──┘└──┘└──┘└──┘└──┘│  ││    │
+        ┌──┐┌──┐┌──┐┌──┐┌──┐└──┘└────┘
+        │  ││  ││  ││  ││  │┌──┐┌────┐
+        └──┘└──┘└──┘└──┘└──┘│  ││    │
+            ┌──┐┌──┐┌──┐┌──┐└──┘└────┘
+            │  ││  ││  ││  │┌──┐┌────┐
+            └──┘└──┘└──┘└──┘│  ││    │
+                    ┌──┐┌──┐└──┘└────┘
+         ┌──┐ ┌──┐  │  ││  │┌──┐┌──┐
+         │  │ │  │  └──┘└──┘│  ││  │
+         │  │ │  │          └──┘└──┘
+         └──┘ │  │
+    ┌──┐ ┌──┐ └──┘
+    │  │ │  │
+    └──┘ └──┘
+     ┌──┐┌──┐
+     │  ││  │
+     └──┘└──┘
+*/
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO,
+                                        KC_NO, KC_NO,
+                                            KC_NO, KC_NO,
+                                            KC_NO, KC_NO,
+//-----------------------------------------------------------------------
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+                KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+                                KC_NO, KC_NO, KC_NO, KC_NO,
+                KC_NO, KC_NO,
+            KC_NO, KC_NO,
+            KC_NO, KC_NO
+    )
```

Comment:
This really should be formatted so that it's side by side, not vertical like this.

Reply:
is there away to do that without reordering the matrix in info.json

Reply:
i think it looks better now.